code/game/machinery/computer/buildandrepair.dm

```diff
@@ -422,6 +422,21 @@
 	var/state = STATE_EMPTY
 	var/obj/item/circuitboard/circuit = null
 
+/obj/structure/computerframe/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It is <b>[anchored ? "bolted to the floor" : "unbolted"]</b>. [anchored ? "" : "Alt-Click to rotate it."]</span>"
```

Comment:
I believe this will have `It is <b>[anchored ? "bolted to the floor` and `Alt-Click to rotate it` on the same line. `Alt-Click to rotate it` should also probably be span classed and at the end of the examine proc for consistency with other `Alt-Click to...`